README.md

```diff
@@ -45,6 +45,8 @@ When executed as `cloud-nuke defaults-aws`, this tool deletes all DEFAULT VPCs a
 3. Add execute permissions to the binary. E.g.: `chmod u+x /usr/local/bin/cloud-nuke`.
 4. Test it installed correctly: `cloud-nuke --help`.
 
+As an alternative, you can install it using [Homebrew](https://brew.sh/): `brew install cloud-nuke`
```

Comment:
NIT: I think we should have our usual caveat about this. See https://terragrunt.gruntwork.io/docs/getting-started/install/#install-via-a-package-manager

Reply:
Not sure if I follow, it's to update the text or add more things?

Reply:
It's to add the paragraph about the homebrew packages being maintained by a 3rd party and not by us, so versions may drift.